recipes/libtiff/all/conanfile.py

```diff
@@ -130,13 +135,23 @@ def generate(self):
         tc.variables["cxx"] = self.options.cxx
         # BUILD_SHARED_LIBS must be set in command line because defined upstream before project()
         tc.cache_variables["BUILD_SHARED_LIBS"] = bool(self.options.shared)
+        tc.cache_variables["CMAKE_FIND_PACKAGE_PREFER_CONFIG"] = True
         tc.generate()
         deps = CMakeDeps(self)
+        if Version(self.version) >= "4.5.1":
+            deps.set_property("jbig", "cmake_target_name", "JBIG::JBIG")
+            deps.set_property("xz_utils", "cmake_target_name", "liblzma::liblzma")
+            deps.set_property("libdeflate", "cmake_file_name", "Deflate")
+            deps.set_property("libdeflate", "cmake_target_name", "Deflate::Deflate")
```

Comment:
Has this changed in new 4.5.1 version? I am curious why previous versions seemed to use other target names, but now changed?

Reply:
> Has this changed in new 4.5.1 version?

In order to reduce patching, this is now dealt with CMakeDeps.set_property rather than introducing a long patch.
All of these were handled by patches in the previous versions and at least the xz_utils target name change which would have required to create a full new patch for dependencies.